.github/workflows/pr-validation-docs.yml

```diff
@@ -0,0 +1,73 @@
+name: Validate Documentation Links
+on:
+  workflow_dispatch:
+    # no content, allows manual triggering
+
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "docs/**"
+
+  repository_dispatch:
```

Comment:
I actually don't think we want/need `ok-to-test` to do anything with this. It can be triggered the same way the `test-generator` job is in `pr-validation`, which is to say, like:

```
  pull_request: 
    branches:
      - main

  merge_group:
    branches:
      - main
```

(with the added docs restriction).

It doesn't need ok-to-test because it doesn't require subscription access.